.github/workflows/test-environment.yml

```diff
@@ -473,6 +474,20 @@ jobs:
           cmd="chmod +x $scriptname && ./$scriptname"
           ../../.ci/scripts/remote_setup.sh -k "$EC2_CSPM_KEY" -s "$src" -h "$CSPM_PUBLIC_IP" -d "~/$scriptname" -c "$cmd"
 
+      - name: Install AWS Asset Inventory integration
+        id: aws-asset-inventory
+        working-directory: ${{ env.INTEGRATIONS_SETUP_DIR }}
+        run: |
+          poetry run python ./install_aws_asset_inventory_integration.py
+
+      - name: Deploy AWS Asset Inventory agent
+        run: |
+          chmod 600 ${{ env.EC2_CSPM_KEY }}
+          # Copy the manifest file to the EC2 instance
+          scp -o StrictHostKeyChecking=no -v -i ${{ env.EC2_ASSET_INV_KEY }} ../../${{ env.INTEGRATIONS_SETUP_DIR }}/aws-asset-inventory-linux.sh "ubuntu@${{ env.ASSET_INV_PUBLIC_IP }}:~/."
+          # Apply the manifest file
+          ssh -o StrictHostKeyChecking=no -v -i ${{ env.EC2_ASSET_INV_KEY }} "ubuntu@${{ env.ASSET_INV_PUBLIC_IP }}" "chmod +x aws-asset-inventory-linux.sh && ./aws-asset-inventory-linux.sh"
+
```

Comment:
```suggestion
          scriptname="aws-asset-inventory-linux.sh"
          src="../../$INTEGRATIONS_SETUP_DIR/$scriptname"
          cmd="chmod +x $scriptname && ./$scriptname"
          ../../.ci/scripts/remote_setup.sh -k "$EC2_ASSET_INV_KEY" -s "$src" -h "$ASSET_INV_PUBLIC_IP" -d "~/$scriptname" -c "$cmd"
```

Reply:
Should I fix it for the entire file, since it's a direct copy of lines above?

Reply:
This should already be fixed throughout the entire file, but if there are any additional occurrences, please apply the fix to those as well